app/pages/UcsfClientEligibilityPage/UcsfClientEligibilityPage.tsx

```diff
@@ -118,62 +125,45 @@ const ClientEligibilities = () => {
 
 const Page = () => {
   const history = useHistory();
-
-  /**
-   * Todo: The below is a general sketch of how we will fetch eligibility data using the category
-   * IDs of the selected UCSF resources. Before we can do this, UCSF resources in our DB will need
-   * to have associated eligibilities and category IDs
-
-    interface resourceListItem {
-      id: string;
-      name: string;
-      icon: string;
-      checked: boolean;
-    }
-
-    // const location = useLocation();
-    interface stateType {
-      selectedResources: resourceListItem[];
-    }
-
-    const { state } = useLocation<stateType>();
-    const selectedResources = state.selectedResources;
-    const resourceEligibilities: object[] = [];
-
-    selectedResources.forEach((resource) => {
-      const eligibilities = useEligibilitiesForCategory(resource.id) || [];
-      resourceEligibilities.push(...eligibilities);
-    });
-  */
-
-  const goToResourceResults = () => {
-    // Todo: We will need to alter the Service Discovery Results file to honor requests
-    // that can include more than one resource (e.g. substance abuse AND mental health)
-    // before successfully navigating to the suggested-resources page
-    alert('Navigate to target resources'); // eslint-disable-line no-alert
-    // history.push('/suggested-resources', {selectedResources: selectedResources});
+  interface LocationState {
+    selectedResourceSlug: string;
+  }
+
+  const state = history.location.state as LocationState;
```

Comment:
You shouldn't need to assert the type on this line if you pass in the state type as a type parameter to `useHistory()`:

```ts
const history = useHistory<LocationState>();
const state = history.location.state;
```